### PR TITLE
fix: ui copy updates

### LINKED ISSUE #7571: Fit & finish - UI Copy | Rocket menu
Fit & finish - UI Copy | Rocket menu

White cells = existing copy
Yellow/blue cells = new copy
Orange = P0

Rocketmenu

Link to 'live' copy on ' Rocketmenu ' tab: 
https://microsoft.sharepoint.com/:x:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B6B1FACBD-B5FE-4AD2-A35B-995635C1696B%7D&file=glossary.xlsx&action=edit&mobileredirect=true&wdPreviousSession=36539654-a42b-4b87-be1b-de96507b4422&wdOrigin=TEAMS-ELECTRON.teams.undefined&cid=676cf454-61e0-4182-bd27-39044bb51cea

### LINKED ISSUE #7572: Fit & finish - UI Copy | Get LUIS
Fit & finish - UI Copy | Get LUIS

White cells = existing copy
Yellow/blue cells = new copy
Orange = P0

Link to 'live' copy on 'Get LUIS ' tab: 
https://microsoft.sharepoint.com/:x:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B6B1FACBD-B5FE-4AD2-A35B-995635C1696B%7D&file=glossary.xlsx&action=edit&mobileredirect=true&wdPreviousSession=36539654-a42b-4b87-be1b-de96507b4422&wdOrigin=TEAMS-ELECTRON.teams.undefined&cid=676cf454-61e0-4182-bd27-39044bb51cea

GetLUIS
Share_resource

### LINKED ISSUE #7585: Fit & finish - UI Copy | Configure and manage your bot (settings)
Fit & finish - UI Copy | Configure and manage your bot (settings)

White cells = existing copy
Yellow/blue cells = new copy (update and replace existing copy with this)
Orange = P0 (must update)

Link to 'live' copy on ' Configure and manage (settings) ' tab: 
https://microsoft.sharepoint.com/:x:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B6B1FACBD-B5FE-4AD2-A35B-995635C1696B%7D&file=glossary.xlsx&action=edit&mobileredirect=true&wdPreviousSession=36539654-a42b-4b87-be1b-de96507b4422&wdOrigin=TEAMS-ELECTRON.teams.undefined&cid=676cf454-61e0-4182-bd27-39044bb51cea

Development
Skills and localization
Connections
Readme

### LINKED ISSUE #7588: Fit & finish - UI Copy | Add KB key, QnA Maker dialogs
Fit & finish - UI Copy | Add KB key

White cells = existing copy
Yellow/blue cells = new copy, copy and replace old copy with this
Orange = Must update (P0)

Link to 'live' copy on 'Add KB key - QnA  ' tab: 
https://microsoft.sharepoint.com/:x:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B6B1FACBD-B5FE-4AD2-A35B-995635C1696B%7D&file=glossary.xlsx&action=edit&mobileredirect=true&wdPreviousSession=36539654-a42b-4b87-be1b-de96507b4422&wdOrigin=TEAMS-ELECTRON.teams.undefined&cid=676cf454-61e0-4182-bd27-39044bb51cea

Add_menu
Add KB wizard
KB progress
Select QnA keys - rocket

### PR DESCRIPTION
**Pull request by benbrown**
## Description

Misc copy changes per @mareekuh's spreadsheets

## Task Item

Fixes #7571 - rocket copy
Fixes #7572 - luis copy
Fixes #7585 - settings copy
Fixes #7588 - qna copy

### CODE REVIEW

File: Composer/packages/client/src/components/GetStarted/GetStartedNextSteps.tsx [hunk 1]
```diff
@@ -265,10 +277,8 @@ export const GetStartedNextSteps: React.FC<GetStartedProps> = (props) => {
       },
       {
         key: 'insights',
-        label: formatMessage('Enable Insights'),
-        description: formatMessage(
-          'Collect service-level and conversation-level data to help gauge the performance and efficacy of your bot.'
-        ),
+        label: formatMessage('Enable App insights'),
```
Comment (hatpick) [inline Composer/packages/client/src/components/GetStarted/GetStartedNextSteps.tsx:280]: nit: App insights OR App Insights?
ref: https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview

File: Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx [hunk 2]
```diff
@@ -50,13 +50,16 @@ export const ManageLuis = (props: ManageLuisProps) => {
     <ManageService
       createService={createService}
       handoffInstructions={formatMessage(
-        '1. Using the Azure portal, please create a Language Understanding resource on my behalf.\n2. Once provisioned, securely share the resulting credentials with me as described in the link below.\n\nDetailed instructions:\nhttps://aka.ms/bfcomposerhandoffluis'
+        '1. Using the Azure portal, please create a Language Understanding resource.\n2. Once created, securely share the resulting credentials with me as described in the link below.\n\nDetailed instructions:\nhttps://aka.ms/bfcomposerhandoffluis'
```
Comment (hatpick) [inline Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx:53]: this link is broken

File: Composer/packages/client/src/components/ManageService/ManageService.tsx [hunk 3]
```diff
@@ -828,6 +828,24 @@ export const ManageService = (props: ManageServiceProps) => {
...
+          setDialogTitle(formatMessage('Select {service} resources', { service: props.serviceName }));
+        } else {
+          setDialogTitle(formatMessage('Create {service} resources', { service: props.serviceName }));
+        }
+        break;
+      case 'resourceCreation':
+        setDialogTitle(formatMessage('Create {service} resources', { service: props.serviceName }));
+        break;
+    }
+  }, [currentStep]);
```
Comment (hatpick) [inline Composer/packages/client/src/components/ManageService/ManageService.tsx:847]: props.serviceName should be added to deps

File: Composer/packages/client/src/pages/botProject/BotProjectSettings.tsx [hunk 4]
```diff
@@ -163,7 +163,7 @@ const BotProjectSettings: React.FC<RouteComponentProps<{ projectId: string; skil
 
   const onRenderHeaderContent = () => {
     return formatMessage(
-      'This Page contains detailed information about your bot. For security reasons, they are hidden by default. To test your bot or publish to Azure, you may need to provide these settings.'
+      "Review and configure your bot's resources, and add new connections such as speech, Microsoft Teams and more."
```
Comment (hatpick) [inline Composer/packages/client/src/pages/botProject/BotProjectSettings.tsx:166]: @beyackle is this ok for localization? (```bot's```) or should it use ```’```

**Review by hatpick — CHANGES_REQUESTED** [review]

File: Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx (hunk 2 above)
Comment (benbrown) [reply Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx:53]: this link is being created by the docs team its on their radar.

**Comment by hatpick** [comment]
@benbrown I forgot, can you run ```yarn l10n``` and include strings (after last comment is resolved)

**Review by hatpick — DISMISSED** [review]

**Review by hatpick — APPROVED** [review]

**Review by hatpick — APPROVED** [review]

**Pull request merged**